docs/data/date-pickers/date-time-picker/date-time-picker.md

```diff
@@ -11,6 +11,12 @@ materialDesign: https://m2.material.io/components/date-pickers
 
 <p class="description">The Date Time Picker component let the user select a date and time.</p>
 
+:::info
+The component by default currently does not ship with time picker view experience on **desktop**.
+It was a conscious decision and a first step towards having a more user friendly desktop experience [discussed in #4483](https://github.com/mui/mui-x/issues/4483).
+If a desktop view experience is essential, it can be reverted to following the suggestion [in the migration guide](/x/migration/migration-pickers-v5/#stop-rendering-a-clock-on-desktop).
```

Comment:
```suggestion
If a desktop view experience is essential, it can be reverted by following the suggestion [in the migration guide](/x/migration/migration-pickers-v5/#stop-rendering-a-clock-on-desktop).
```

Reply:
🙈 We probably need both in this case.
What do you think about these options?
- If a desktop view experience is essential, it can be reverted by following the suggestion in the migration guide.
- If a desktop view experience is essential, you can revert to it by following the suggestion in the migration guide.

`You can` is used in our docs quite often, maybe it sounds a bit more correct that way? 🤔

Reply:
I prefer the `you can` :+1:

Reply:
Done. 👌